miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/ads/AdMob20Displayer.kt

```diff
@@ -0,0 +1,124 @@
+package com.rakuten.tech.mobile.miniapp.ads
+
+import android.app.Activity
+import android.util.Log
+import com.rakuten.tech.mobile.admob.AdmobDisplayerLatest
+import com.rakuten.tech.mobile.admob.AdStatus
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
+import kotlin.coroutines.CoroutineContext
+
+/** Check whether hostapp provides latest admob dependency. */
+@Suppress("EmptyCatchBlock", "SwallowedException")
+internal inline fun <T> whenHasAdmobLatest(callback: (isAvailable: Boolean) -> T) {
+    try {
+        Class.forName("com.rakuten.tech.mobile.admob.AdmobDisplayerLatest")
+        callback.invoke(true)
+    } catch (e: ClassNotFoundException) {
+        callback.invoke(false)
+    }
+}
+
+/**
+ * The ad displayer for latest admob sdk 20.2.0.
+ * @param context should use the same activity context for #MiniAppDisplay.getMiniAppView.
+ * Support Interstitial, Reward ads.
+ */
+class AdMob20Displayer(private val context: Activity) : MiniAppAdDisplayer, CoroutineScope {
```

Comment:
See my other comments: basically this class should exist in the `admob-latest` module. If the Host App needs to use Ads SDK 20+, then they should import this class from that module.